src/win32.mak

```diff
@@ -308,6 +321,8 @@ clean:
 	$(DEL) impcnvtab.c
 	$(DEL) id.h id.c
 	$(DEL) verstr.h
+	$(DEL) $(GENSRC)
+	$(DEL) $(MAGICPORT)
```

Comment:
These cleanups in win32.mak don't seem to work, they just produce a `del`command without arguments. (because the macros are defined after the command!?). The libraries are also missing, but that's not new in this PR.